package.json

```diff
@@ -76,10 +76,6 @@
       ],
       "category": "Development"
     },
-    "directories": {
-      "buildResources": "resources",
-      "output": "release"
```

Comment:
I think we are depending on this directory being named `release` in our GitHub workflows.

Reply:
You will need to adapt them :)